Assets/HoloToolkit-Examples/ARCA/Scripts/ColorChanger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class ColorChanger : NetworkBehaviour
+
+{
+    [SyncVar]
+    Color color;
```

Comment:
fields, properties and methods should all have explicit access specifiers (private here). Also, please add summary comments describing the purpose / function of each field, method, etc.